### ISSUE TITLE
gql_build  is incompatible with freezed

### ISSUE THREAD

**Issue by EhabSalah** [gql-dart/gql#275]
## Am facing this error while building the project

```Because gql_build >=0.2.0-alpha+1621196644782 <0.3.0-alpha+1632808907187 depends on analyzer ^1.2.0 and freezed ^0.14.3 depends on analyzer ^2.0.0, gql_build >=0.2.0-alpha+1621196644782 <0.3.0-alpha+1632808907187 is incompatible with freezed ^0.14.3.```

**Comment by adiospace** [comment]
Is it working fine for you guys?

**Comment by diegog-sf** [comment]
Any news regarding this? we're already on `analyzer 4.0`

**Comment by vast00** [comment]
Same here,  any news?

`
Because gql_build >=0.4.0 depends on analyzer ^2.3.0 and freezed ^1.1.1 depends on analyzer ^3.0.0, gql_build >=0.4.0 is incompatible with freezed ^1.1.1.
So, because dotai_client depends on both freezed ^1.1.1 and gql_build ^0.4.0, version solving failed.
pub get failed (1; So, because dotai_client depends on both freezed ^1.1.1 and gql_build ^0.4.0, version solving failed.)
`

**Comment by EhabSalah** [comment]
```
[WARNING] ../../../../../.pub-cache/hosted/pub.dartlang.org/gql_build-0.2.0/lib/src/serializer_builder.dart:69:30: Error: The getter 'types' isn't defined for the class 'CompilationUnitElement'.
 - 'CompilationUnitElement' is from 'package:analyzer/dart/element/element.dart' ('../../../../../.pub-cache/hosted/pub.dartlang.org/analyzer-4.0.0/lib/dart/element/element.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'types'.
          .expand((cu) => cu.types)
                             ^^^^^
```

**Comment by knaeckeKami** [comment]
New alpha releases for gql_build, gql_code_builder and ferry_generator have been released with support for analyzer 4.0.0.

I will work on stable releases soon.
For now, you can either use the alpha versions or pin freezed to 2.0.3.

**Comment by knaeckeKami** [comment]
New version was released.
